src/constants.ts

```diff
@@ -40,6 +40,7 @@ export const PERCENTAGE_SIG_DIGITS = 4;
 export const PROTOCOL_FEE_UTILS_POLLING_INTERVAL_IN_MS = 6000;
 export const UNWRAP_QUOTE_GAS = new BigNumber(60000);
 export const WRAP_QUOTE_GAS = UNWRAP_QUOTE_GAS;
+export const AFFILIATE_FEE_TRANSFORMER_GAS = new BigNumber(10000);
```

Comment:
probably would be a bit more than 10k?

Reply:
probably closer to 20k if the recipient doesn't have a balance yet, but they probably do –– idk @dorothy-zbornak does 10k sound reasonable here?

Reply:
Let's be conservative and go 15k for USDC.